backend/dashboard_metrics/services.py

```
@@ -26,6 +26,19 @@
 from unstract.core.data_models import ExecutionStatus
 
 
+def _get_hitl_queue_model():
+    """Get HITLQueue model if available (cloud-only).
+
+    Returns None on OSS where manual_review_v2 is not installed.
+    """
+    try:
+        from pluggable_apps.manual_review_v2.models import HITLQueue
+
+        return HITLQueue
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)